data/coalition/coalition ships.txt

```diff
@@ -695,6 +1150,175 @@ ship "Heliarch Punisher" "Heliarch Punisher (Scrappy)"
+	gun 0 -35.5
+	explode "tiny explosion" 20
+	explode "small explosion" 10
+	description `Occasionally, when a Heliarch cadet shows promising results in dogfighting simulations, they are allowed to fly a ship of their own, that being the Pursuer.`
+	description `	Though certainly not as durable as the other fighters in Coalition space, this Kimek design is cheaper to produce, and is capable of outrunning any other Coalition ship.`
+
+ship "Heliarch Pursuer" "Heliarch Pursuer (Scrappy)"
+	outfits
+		"Bombardment Cannon"
+		
+		"Large Collector Module"
+		"Small Collector Module" 3
+		"Small Battery Module"
+		"Small Shield Module"
+		"Small Repair Module"
+		
+		"Small Thrust Module" 2
+		"Small Steering Module" 2
+
+
+ship "Heliarch Rover"
+	sprite "ship/heliarch rover"
+	thumbnail "thumbnail/heliarch rover"
+	attributes
+		category "Fighter"
+		licenses
+			Heliarch
+		"cost" 1238000
+		"shields" 3300
+		"hull" 3100
+		"required crew" 1
+		"bunks" 1
+		"mass" 37
+		"drag" 1.19
+		"heat dissipation" .96
+		"outfit space" 125
+		"weapon capacity" 24
+		"engine capacity" 32
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Bombardment Cannon" 2
+		
+		"Large Collector Module"
+		"Large Battery Module"
+		"Small Shield Module"
+		"Small Repair Module" 3
+		
+		"Small Thrust Module" 2
+		"Small Steering Module" 2
+	
+	engine 7 28
+	engine -7 28
+	gun 5 -30
+	gun -5 -30
+	explode "tiny explosion" 25
+	explode "small explosion" 15
+	description `The very first Fighter the Heliarchs designed, Rovers are well-rounded ships that the Heliarchs employ regularly due to their reliability in any situation.`
+
+ship "Heliarch Rover" "Heliarch Rover (Ions)"
+	outfits
+		"Ion Rain Gun"
+		
+		"Large Collector Module"
+		"Small Collector Module" 3
+		"Small Battery Module"
+		"Small Shield Module"
+		"Large Repair Module"
+		
+		"Small Thrust Module" 2
+		"Small Steering Module" 2
+
+
+ship "Heliarch Stalker"
+	sprite "ship/heliarch stalker"
+	thumbnail "thumbnail/heliarch stalker"
+	attributes
+		category "Fighter"
+		licenses
+			Heliarch
+		"cost" 1358000
+		"shields" 5700
+		"hull" 5200
+		"required crew" 1
+		"bunks" 2
+		"mass" 80
+		"drag" 2.1
+		"heat dissipation" 0.64
+		"outfit space" 133
+		"weapon capacity" 43
+		"engine capacity" 16
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Ion Hail Turret"
+		
+		"Large Collector Module"
+		"Small Collector Module"
+		"Small Battery Module"
+		"Large Shield Module"
+		"Large Repair Module"
+		
+		"Small Steering Module"
+		"Small Thrust Module"
+	
+	engine 0 40 .8
+	turret 0 -1.5
+	explode "tiny explosion" 25
+	explode "small explosion" 15
+	description `Boasting incredible durability for a ship this size, the Stalker is a compact fortress that can take a lot of punishment and recover faster than some warships. Its speed and maneuverability, however, are all but impressive.`
```

Comment:
```suggestion
	description `Boasting incredible durability for a ship this size, the Stalker is a compact fortress that can take serious punishment and recover faster than most other warships. Its speed and maneuverability, however, leave much to be desired.`
```
Just style.

Reply:
outside of human warships (that aren't using the S-series Regenerators), almost all other warships in the game have more regen than the Stalker's, so I left it at `some` instead of swapping for `most other`